srtcore/threadname.h

```diff
@@ -68,8 +76,28 @@ class ThreadName
 class ThreadName
 {
 public:
+    static const size_t BUFSIZE = 128;
 
-    static bool get(char*) { return false; }
+    static bool get(char* output)
+    {
+        // The default implementation will simply try to get the thread ID
+#if defined(__cplusplus) && __cplusplus > 199711L // <-- HAVE_C++11
```

Comment:
This should probably go in align with `USE_STDCXX_CHRONO` definition for `sync` functionality.

Reply:
It could, although it's simply in order to provide any kind of "thread ID" - no matter if the current implementation is using this thread library or not.

OTOH, users might resign C++11 threads just because they have problems with linking against some parts of C++ library, and this would work against them.

This may even be postponed up to when whole C++11 support is in place, and there could be some better defs for it.